pkg/controller/plan/kubevirt.go

```diff
@@ -499,6 +500,14 @@ func (r *KubeVirt) EnsureDataVolumes(vm *plan.VMStatus, dataVolumes []cdi.DataVo
 				"vm",
 				vm.String())
 		}
+		pvcNames = append(pvcNames, dv.Name)
+	}
+	// DataVolume and PVC name are the same
```

Comment:
this comment is supposed to be next to `pvcNames = append(pvcNames, dv.Name)`, no?

Reply:
yup :)